### ISSUE TITLE
Unique commit messages

### ISSUE THREAD

**Issue by maltsev** [nikitavoloboev/gitupdate#15]
If I'm working on the same files and doing these temp commits, I guess the commit message would always be the same? It'd be awesome to add some uniqueness to commit messages to easily differentiate them. For example:

```
5dcc60a0 my-cool-script 👝 🌴 😠
331c837a my-cool-script 📂 👉 😜
1d7d10a4 my-cool-script 🔅 😻 🚶
bbdfb9a my-cool-script 🐅 🙉 🎣
612a7d6 my-cool-script
```

**Comment by nikitavoloboev** [comment]
> add some uniqueness to commit messages to easily differentiate them

Random emojis won't be adding any new information though, I think. Or I am missing something?

**Comment by maltsev** [comment]
They'll help differentiate between commits with the same name. For example, when doing rebasing or just referring to a commit by its name.

Actually, maybe instead of random emojis, it'd be better to use ordinal numbers? So it's clear which commit has been created first.

**Comment by chapmanjacobd** [comment]
yeah incrementing numbers in parentheses if the commit message is the same as the last commit message (removing all parentheses) seems like a useful feature

**Comment by chapmanjacobd** [comment]
With some extra time on my hands I made something a little more descriptive. If you're just modifying one file it will include the edit that you made into the commit message, or if too long, count the number of added, deleted, and moved lines:

https://github.com/chapmanjacobd/computer/blob/main/bin/wip_message.py

**Comment by nikivdev** [comment]
Super cool, thanks for sharing it
